packages/@ember/-internals/glimmer/lib/helpers/fn.ts

```diff
@@ -6,6 +6,52 @@ import { InternalHelperReference } from '../utils/references';
 import buildUntouchableThis from '../utils/untouchable-this';
 
 const context = buildUntouchableThis('`fn` helper');
+
+/**
+@module ember
+*/
+
+/**
+  The `fn` helper is used to wrap the provided callback function up with a
+  set of arguments. When the returned function is invoked the callback will
+  receive the arguments from the template along with the arguments passed when
+  invoking the returned function. Importantly, the `fn` helper does **not**
```

Comment:
"When the returned function is invoked the callback will receive the arguments from the template along with the arguments passed when invoking the returned function."

Can you state this another way, or in more words? I am not sure what this sentence means. It sounds like there are two sets of arguments that it gets, but I only see one arg in the example.